### PR TITLE
Adds english translations #37

### PR DESCRIPTION
**Pull request by iamfelipemattos**
**CHANGELOG** :memo:
- Added 
  -  [x]  [ Front End Engineer - Junior ] session

### CODE REVIEW

File: translations/en/front-end-engineer/README.md [hunk 1]
```diff
@@ -1 +1,24 @@
-# Front End Engineer path
\ No newline at end of file
+# Career levels the a Front End Engineer
```
Comment (effexts) [inline translations/en/front-end-engineer/README.md]: "the a" this doesn't exist. The correct sentence would be like "Front-End Engineer: Career Levels"

File: translations/en/front-end-engineer/README.md [hunk 2]
```diff
@@ -1 +1,24 @@
...
+
+## Junior Front End Engineer 
+
+Now what you know whon your responsibility like Junior Developer, time has come to focus us technical requirements.
```
Comment (effexts) [inline translations/en/front-end-engineer/README.md]: This line is bad translated too.
The correct one should be: "Now, that you know which are your responsibilities as Junior Developer", it's time to focus on..."

File: translations/en/front-end-engineer/README.md [hunk 3]
```diff
@@ -1 +1,24 @@
...
+
+Requirements for Junior Front End Engineer:
+
+* HTML (encode a basic struct, understanding about semantic and accessibility)
```
Comment (effexts) [inline translations/en/front-end-engineer/README.md]: struct is different than structure. Here goes the word "structure".

File: translations/en/front-end-engineer/README.md [hunk 4]
```diff
@@ -1 +1,24 @@
...
+* The Basic responsive (learn do make a layout work in multiple devices) 
+* Git (And know that Git and Github not are the same thing)
+
+Depending the his zone/company where working you gonna need know both to handle like any edit image software (aka Photoshop). For know if really need, of a good search in vacancies local
+
+not run during the studies, a lot practice all the you learn until have courage in speak what know that!
+
+Case you dominates already this things, you maybe already be a Mid Level Front End Engineer
+
+if you are seeking an study source take a [look here](../study-guides/README.md)
```
Comment (effexts) [inline translations/en/front-end-engineer/README.md]: Every sentence should start with a capital letter.

**Comment by effexts** [comment]
This one has a lot of mistakes. Please check.

**Comment by iamfelipemattos** [comment]
sorry, I'm still learning.

**Comment by iamfelipemattos** [comment]
@effexts, take a look in my corrections.

Thank you very much for correcting my mistakes

File: translations/en/front-end-engineer/README.md [hunk 5]
```diff
@@ -1,12 +1,12 @@
...
 
 ## Junior Front End Engineer 
 
-Now what you know whon your responsibility like Junior Developer, time has come to focus us technical requirements.
+Now, that you know which are your responsibilities as Junior Developer, it's time to focus on technical requirements.
 
 Requirements for Junior Front End Engineer:
 
-* HTML (encode a basic struct, understanding about semantic and accessibility)
+* HTML (encode a basic structure, understanding about semantic and accessibility)
```
Comment (effexts) [inline translations/en/front-end-engineer/README.md]: change encode for code.
change "understanding" for understand

File: translations/en/front-end-engineer/README.md [hunk 6]
```diff
@@ -1,12 +1,12 @@
...
 * CSS ("coding" a layout, to get copy any website or simple component is a good indicator of your knowledge level)
```
Comment (effexts) [inline translations/en/front-end-engineer/README.md]: "coding" for code, and you lost the links, fix that.
remove "to get"

**Comment by woliveiras** [comment]
Thanks by helping @IAMFELIPEMATTOS, @effexts 

My english is GTO (Google Translate oriented), so I can not helping very much here! :joy:

**Pull request merged**